doc/manuals/contribution_guidelines.md

```diff
@@ -750,6 +750,14 @@ void myFunction(const std::string s, std::string* err);
 
 *How to check*: manually
 
+#### S16 (Use empty() for checking zero-length strings)
+
+*Rule*: Use empty() for all checkings with "" and usages of length()
+
+*Rationale*: empty() is the best way of checking for zero-length strings.
```

Comment:
```suggestion
*Rationale*: empty() is [the best way of checking for zero-length strings](http://stackoverflow.com/questions/41425569/checking-for-empty-string-in-c-alternatives).
```

To include a link to the place were the discussion took place.

Reply:
Updated